paper/paper.md

```diff
@@ -24,19 +24,19 @@ bibliography: paper.bib
 
 # Summary
 
-Spherical Harmonic Transforms (SHTs) can be seen as the spherical, two-dimensional, counterpart of Fourier Transforms, casting real-space data to the spectral domain and vice versa.
-As in Fourier analysis a function is decomposed into a set of amplitude coefficients, through an SHT, any spherically-symmetric field defined in real space can be decomposed into a set of complex harmonic coefficients $a_{\ell, m}$, commonly referred to as alms, each quantifying the contribution of the corresponding spherical harmonic function.
+Spherical Harmonic Transforms (SHT) can be seen as Fourier Transforms' spherical, two-dimensional counterparts, casting real-space data to the spectral domain and vice versa.
+As in Fourier analysis, a function is decomposed into a set of amplitude coefficients through an SHT, and any spherically-symmetric field defined in real space can be decomposed into a set of complex harmonic coefficients $a_{\ell, m}$, commonly referred to as alms, each quantifying the contribution of the corresponding spherical harmonic function.
```

Comment:
I think this rephrasing slightly changes the meaning of the sentence. I would keep the original.

Reply:
just be careful with the plural. I think "counterpart`s`" is grammatically correct here, isn't it?

Reply:
> just be careful with the plural. I think "counterpart`s`" is grammatically correct here, isn't it?

Yes you're most probably right. Then I think we can accept the changes for line 27 while keeping line 28 as it is. What do you think?